source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -207,24 +225,21 @@ def _filter_similar_journeys(journey_pairs_pool, request, similar_journey_genera
                           .format(other=j1.internal_id if worst == j2 else j2.internal_id))
 
 
-def exceed_min_duration(current_section, journey, request,  min_duration):
+def exceed_min_duration(current_section, journey, min_duration, orig_modes=[], dest_modes=[]):
```

Comment:
![trap](https://user-images.githubusercontent.com/6093486/41342515-b5884978-6efc-11e8-83fd-0c9625e9239c.gif)

:laughing: 

what are you expecting this code to print?

```python
def f(l=[]):
    l.append('a')
    print(l)

f()
f()
f()
```

Reply:
@xlqian : :scream: this is evil !